src/main.ts

```diff
@@ -1,6 +1,9 @@
 import { ValidationPipe } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import * as dotenv from 'dotenv';
```

Comment:
not necessary since nestjs has its own environment library, you can read about it [here](https://docs.nestjs.com/techniques/configuration)